service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/GeneratedOrderController.java

```diff
@@ -165,6 +169,6 @@ private String concatGatewayConfigurationUrlAndMostRecentUploadedOrderDocumentPa
     }
 
     private DocmosisTemplates getDocmosisTemplateType(GeneratedOrderType type) {
-        return type == EMERGENCY_PROTECTION_ORDER ? EPO : ORDER;
+        return type == EMERGENCY_PROTECTION_ORDER ? EPO : COVER_SHEET;
```

Comment:
Hack - Will be reverted. Using the orders mid event to generate the COVER_SHEET for QA.